blues/simulation.py

```diff
@@ -423,49 +442,47 @@ def acceptRejectNCMC(self, temperature=300, write_move=False, **opt):
     def simulateNCMC(self, nstepsNC=5000, ncmc_traj=None,
                     reporter_interval=1000, verbose=False, **opt):
         """Function that performs the NCMC simulation."""
-
-        self.log.info('[NCMC-Iter %i] Advancing %i NCMC steps...' % (self.current_iter, nstepsNC))
-
+        self.log.info('[Iter %i] Advancing %i NCMC steps...' % (self.current_iter, nstepsNC))
         #choose a move to be performed according to move probabilities
-        #TODO: will have to change to work with multiple alch regions
+        #TODO: will have to change to work with multiple alch region
         self.move_engine.selectMove()
+        move_idx = self.move_engine.selected_move
```

Comment:
What are we changing here? not totally obvous to me.

Reply:
No changes here, just grabbing the move index for printing which move type is being applied. This will be useful when we are ready for mixing move types